server/src/main/java/org/elasticsearch/action/support/IndicesOptions.java

```diff
@@ -446,6 +449,14 @@ public static IndicesOptions strictExpand() {
         return STRICT_EXPAND_OPEN_CLOSED;
     }
 
+    /**
+     * @return indices option that requires every specified index to exist, expands wildcards to both open and closed
+     * indices and allows that no indices are resolved from wildcard expressions (not returning an error).
```

Comment:
Maybe add that data streams are included to distinguish it from `::strictExpand()`